gdal/doc/source/drivers/vector/elasticsearch.rst

```diff
@@ -279,7 +279,8 @@ options:
    will be written as GeoJSON Feature objects. If another mapping name
    is chosen, a more "flat" structure will be used.
 -  **MAPPING**\ =filename or JSon. Filename from which to read a
-   user-defined mapping, or mapping as serialized JSon.
+   user-defined mapping, or mapping as serialized JSon. This option is
+   ignored when converting to Elasticsearch >=7 (see `Removal of mapping types <https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html`__).
```

Comment:
causes `/__w/1/s/gdal/doc/source/drivers/vector/elasticsearch.rst:: WARNING: Anonymous hyperlink mismatch: 1 references but 0 targets.`

Reply:
😞 Sorry, about that. I've pushed a fix after testing the doc build.